MediaBrowser.XbmcMetadata/Savers/BaseNfoSaver.cs

```diff
@@ -25,76 +25,81 @@ namespace MediaBrowser.XbmcMetadata.Savers
 {
     public abstract class BaseNfoSaver : IMetadataFileSaver
     {
-        public static readonly string YouTubeWatchUrl = "https://www.youtube.com/watch?v=";
-
-        private static readonly CultureInfo UsCulture = new CultureInfo("en-US");
-
-        private static readonly Dictionary<string, string> CommonTags = new[] {
-
-                    "plot",
-                    "customrating",
-                    "lockdata",
-                    "dateadded",
-                    "title",
-                    "rating",
-                    "year",
-                    "sorttitle",
-                    "mpaa",
-                    "aspectratio",
-                    "collectionnumber",
-                    "tmdbid",
-                    "rottentomatoesid",
-                    "language",
-                    "tvcomid",
-                    "tagline",
-                    "studio",
-                    "genre",
-                    "tag",
-                    "runtime",
-                    "actor",
-                    "criticrating",
-                    "fileinfo",
-                    "director",
-                    "writer",
-                    "trailer",
-                    "premiered",
-                    "releasedate",
-                    "outline",
-                    "id",
-                    "credits",
-                    "originaltitle",
-                    "watched",
-                    "playcount",
-                    "lastplayed",
-                    "art",
-                    "resume",
-                    "biography",
-                    "formed",
-                    "review",
-                    "style",
-                    "imdbid",
-                    "imdb_id",
-                    "country",
-                    "audiodbalbumid",
-                    "audiodbartistid",
-                    "enddate",
-                    "lockedfields",
-                    "zap2itid",
-                    "tvrageid",
-
-                    "musicbrainzartistid",
-                    "musicbrainzalbumartistid",
-                    "musicbrainzalbumid",
-                    "musicbrainzreleasegroupid",
-                    "tvdbid",
-                    "collectionitem",
-
-                    "isuserfavorite",
-                    "userrating",
-
-                    "countrycode"
-
-        }.ToDictionary(i => i, StringComparer.OrdinalIgnoreCase);
+        public const string DateAddedFormat = "yyyy-MM-dd HH:mm:ss";
+
+        public const string YouTubeWatchUrl = "https://www.youtube.com/watch?v=";
+
+        private static readonly CultureInfo _usCulture = CultureInfo.ReadOnly(new CultureInfo("en-US"));
```

Comment:
This should probably be replace by InvariantCulture, just like we did in other spots in the code.